staker/block_validator.go

```diff
@@ -796,6 +841,13 @@ func (v *BlockValidator) iterativeValidationPrint(ctx context.Context) time.Dura
 	}
 	log.Info("validated execution", "messageCount", printedCount, "globalstate", validated.GlobalState, "WasmRoots", validated.WasmRoots)
 	v.lastValidInfoPrinted = validated
+	if v.config().Dangerous.Revalidation.EndBlock > 0 && validated.GlobalState.Batch >= v.config().Dangerous.Revalidation.EndBlock {
```

Comment:
nitpick: config().Dangerous.Revalidation can change during the execution of this function, since BlockValidatorConfig is a hot config.
It would be nice to read Revalidation config into memory and use it instead of always relying on config()

Reply:
Done.